### ISSUE TITLE
Stub event bus/broadcasting package

### ISSUE THREAD

**Issue by brandonjpierce** [gohypergiant/standard-toolkit#17]
#### Must haves:
 
- Cross context comms (main thread, web worker, shared worker)
- Context isolation (main thread only, or this tab only)
- Decoupled from any form of persistence
- Strong type safety
- Strong debug tooling / observability
 
---
 
#### Nice to haves
 
- Interop with redux dev tools for introspection
 
---
 
#### Applicable areas to start with:
 
- Offscreen indicators
- Map interaction (zoom to, tether, center on)
- Baseball card management (get me out of React render cycle driving baseball cards)
- Layer interaction (click / hover layer)
- Layer management (hide/show layer, change layer order)

**Comment by brandonjpierce** [comment]
Yeah I think we need to chat with the folks that did some micro RfC's like Andrew and bring that into the larger conversation as you noted before we make a decision.

**Comment by belsrc** [comment]
Since Andrew is already taken this up in another repo, and will likely be in charge of bringing it over to here, do you want to close this issue?

**Comment by brandonjpierce** [comment]
Yep good to close 👍
